MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1952,6 +1952,23 @@
 #endif
 
 #endif  // MVK_MACOS
+
+#if MVK_XCODE_16
+	if (_isUsingMetalArgumentBuffers && _metalFeatures.residencySets) {
+		MTLResidencySetDescriptor *setDescriptor;
+		setDescriptor = [MTLResidencySetDescriptor new];
```

Comment:
No biggie, but I'm never a fan of uninitialized vars. It would read nicer if these two lines were combined.